database/migrations/0068-community_tables_public_key_length.ts

```diff
@@ -39,4 +39,4 @@ export async function downgrade(queryFn: (query: string, values?: any[]) => Prom
   await queryFn(
     'ALTER TABLE `communities` MODIFY COLUMN `public_key` binary(64) NULL DEFAULT NULL;',
   )
-}
\ No newline at end of file
+}
```

Comment:
Changed Line Feed? From Unix Type to Windows Type?

Reply:
No the linter wants an additional line at the end of file.
An i checked the linefeeds, they are in unix format...